packages/ra-ui-materialui/src/input/ReferenceArrayInput.stories.tsx

```diff
@@ -70,6 +70,68 @@ export const Basic = () => (
     </Admin>
 );
 
+export const ReadOnly = () => (
+    <Admin dataProvider={dataProvider} history={history}>
+        <Resource name="tags" recordRepresentation={'name'} />
+        <Resource
+            name="posts"
+            create={() => (
+                <Create
+                    resource="posts"
+                    record={{ tags_ids: [1, 3] }}
+                    sx={{ width: 600 }}
+                >
+                    <SimpleForm>
+                        <ReferenceArrayInput
+                            reference="tags"
+                            resource="posts"
+                            source="tags_ids"
+                            readOnly
+                        />
+                        <ReferenceArrayInput
+                            reference="authors"
+                            resource="name"
+                            source="authors_ids"
+                            disabled
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** ‏

```suggestion
                            readOnly
```